MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -336,7 +336,7 @@ public IUser FindUser(Guid id)
 
         internal void OnReceive(Message message)
         {
-            if (message.Payload is LoadAssets || message.Payload is AssetUpdate)
+            if (message.Payload is LoadAssets || message.Payload is CreateAsset || message.Payload is AssetUpdate)
```

Comment:
This pattern isn't scaling well. Will have to find a better way eventually.

Reply:
Agreed. If I have to add one more message, I'll make a types array.